src/pages/home/report/ReportActionContextMenu.js

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import {Pressable, View} from 'react-native';
+import PropTypes from 'prop-types';
+import {
+    Clipboard, LinkCopy, Mail, Pencil, Trashcan,
+} from '../../../components/Icon/Expensicons';
+import styles from '../../../styles/styles';
+import getReportActionContextMenuStyles from '../../../styles/getReportActionContextMenuStyles';
+import getButtonState from '../../../libs/getButtonState';
+import Icon from '../../../components/Icon';
+import Tooltip from '../../../components/Tooltip';
+
+const CONTEXT_ACTIONS = [
+    // Copy to clipboard
+    {
+        text: 'Copy to Clipboard',
+        icon: Clipboard,
+    },
+
+    // Copy chat link
+    {
+        text: 'Copy Link',
+        icon: LinkCopy,
+    },
+
+    // Mark as Unread
+    {
+        text: 'Mark as Unread',
+        icon: Mail,
+    },
+
+    // Edit Comment
+    {
+        text: 'Edit Comment',
+        icon: Pencil,
+    },
+
+    // Delete Comment
+    {
+        text: 'Delete Comment',
+        icon: Trashcan,
+    },
+];
+
+const propTypes = {
+    // The ID of the report this report action is attached to.
+    // eslint-disable-next-line react/no-unused-prop-types
+    reportID: PropTypes.number.isRequired,
```

Comment:
Where is this used? 🤔

Reply:
It's not yet, but remember that the base branch of this PR is a feature branch, and this field will be used by all the context actions (edit report, mark as read, etc...)